src/renderer/screens/Operations/common/utils.ts

```diff
@@ -219,3 +222,23 @@ export const getTransactionAmount = (tx: Transaction): string | null => {
 
   return null;
 };
+
+export const getSignatoryName = (
```

Comment:
Better use domain types, there are no `id's` down below
```suggestion
export const getSignatoryName = (
  signatoryId: AccountId,
  tx: MultisigTransaction,
  contacts: Contact[],
  accounts: Account[],
  addressPrefix?: number,
): string => {
```